src/Tribe/Views/V2/View.php

```diff
@@ -534,8 +536,12 @@ public function get_html() {
 		 */
 		$this->repository_args = $repository_args;
 
+		// If HTML_Cache is a class trait and we have content to display, display it.
+		if ( method_exists( $this, 'maybe_get_cached_html' ) && $cached_html = $this->maybe_get_cached_html() ) {
```

Comment:
:no_entry_sign: Assignments must be the first block of code on a line

<sub>hash: e6839661bad94eabad610a251a539300</sub>